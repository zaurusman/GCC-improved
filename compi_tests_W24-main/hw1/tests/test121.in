mIcmW2IPgoJGF9N"\\ \t 1670 89464 \x30 ==truenotIzOF4ROVNi29VtOxaJ"N 61016 cdZFaHipo <=R6T1y
*kdaWZn"\0 F r 11132 \\\n\x3E }QGYG "A \\ 29671m i2 \r "!="\0 JdxGMf6jzfG \x6E796 \r 66620 p wcpBkdYq45X \B 68928 \i\a EI6 \t \r e 
}bool!=(ypUWVf8ymintiffalseif
orJUZvPd7hPJUZvPd7hPog8mvlSsVHqH=/notSUljGeRW8VJ"\t \R \n \\\ \0 \x56 x XejCxjlzAPWztV E\x08 qjtECJxIhKTJ6 CBdo585PBB ;"\r\t \0 \r z 31432 \" \\ \0 \\\t \\ lb8Jrw \0 A %\r 92910 xrGiLkWuQP4zMH ("y \\\ \n \0 zPWvb4GuHDWuiqIvoidnot"\\\ \0 \r d \r \t d 78850 \x4C FLMAHn1C016944 52869 wNkGH \r 34496I b{orytiePqGLor==8r3qQdChwhybpl"p 28502 \x62 \x65 Y 22015 \" 13026 50631 w c0U \x4B 011173 RNT3AZ7kZrk39hQ \x20 "94720 37856 zR "p 28502 \x62 \x65 Y 22015 \" 13026 50631 w c0U \x4B 011173 RNT3AZ7kZrk39hQ \x20 "\\\ \\ \" \f \0 \t 6tKVHyz7LK3MnSS0}
"\n \r fwBzR \\\ \0 x 22453 \N \\ bvXR \r \\\ r \t 521440qhs B111qjTVNQMsUYm orbyte"\\\ \0 \x68 "\\ z \\ r \x35 55862 17189 nCrMCqLZGnY9Ahu"\\\ \0 \x68 uIJn2"C70195LK7NCTX86Rc false
6johXmIWbvoidand)Vf7sbLoe3iftrue"\" KIIyV H \n \\\ 79721 Tssfq4KQ0NW-true{whileint +b"eg4ktcXv "lJ7PEkgegalM B W8m5oar G 0qZH5\t x \\ 57542 cKijtlwnv2X3nyi076iKXQG5Y O eP8I \\\ \n \" F 
"2 \x28 0036829 \0 \" \\\ \n 18457 89530 OsPBV \t \\ 0090176 \rT \nfalse} false"O w & \\\oqxpTRkR Cm \xA1\T n Q pj U 54371 L 60485 hkUyl4 "\0 \l\r k1F6F5A3waA\n \0 \" \r \t 60723 a \x47 \\\>=truesgWKg
