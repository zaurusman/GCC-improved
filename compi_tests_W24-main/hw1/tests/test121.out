1 ID mIcmW2IPgoJGF9N
1 STRING \ 	 1670 89464 0 ==truenotIzOF4ROVNi29VtOxaJ
1 ID N
1 NUM 61016
1 ID cdZFaHipo
1 RELOP <=
1 ID R6T1y
2 BINOP *
2 ID kdaWZn
2 STRING 
2 ID A
Error \
